### ISSUE TITLE
HID mouse / keyboard interface emulator driver for Windows/Mac

### ISSUE THREAD

**Issue by shawaj** [PiSupply/Flick#3]
As above, a driver for Windows/Mac OS using an [I2C to USB bridge](http://uk.farnell.com/microchip/adm00559/evaluation-board-usb-to-uart-i2c/dp/2450806) that lets you connect Flick over I2C and control the mouse and some additional KB events.

**Comment by tvoverbeek** [comment]
Instead of an I2C to USB bridge probably easier to take something like an ATmega32u4 to emulate a USB keyboard/mouse.
Would also be an option for Linux

**Comment by shawaj** [comment]
Yeah that sounds like a great idea too. Only reason I suggested USB to i2c is because I believe a driver already exists for that that could be tweaked
